pkg/fileutils/fileutils.go

```diff
@@ -1,33 +1,105 @@
 package fileutils
 
 import (
+	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/Sirupsen/logrus"
 )
 
+func Exclusion(pattern string) bool {
+	return pattern[0] == '!'
```

Comment:
not sure that it will hit here, but the callers of this look to have used `filepath.Clean()` which does nothing for leading (or trailing) whitespaces. So `Exclusion("  !foo  ")` would return `false`